spec/lib/account_deleter_spec.rb

```diff
@@ -92,7 +92,7 @@
   end
 
   describe "#delete_standard_user_associations" do
-    it 'removes all standard user associaltions' do
+    it 'removes all standard user associations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.